.github/workflows/playwright.yml

```
@@ -32,14 +98,36 @@
+        run: |
+          if [ ! -d ".next" ] || [ -z "$(ls -A .next 2>/dev/null)" ]; then
+            echo "Build artifacts not found, building locally..."
+            npm run build
+          else
+            echo "Using cached build artifacts"
+          fi
       - name: Run Playwright tests (shard 1 of 2)
         run: |
-          npm run build
-          NEXTAUTH_SECRET=SECRET npm start & npx playwright test --grep-invert "@config|@tls|@cluster" --shard=1/2 --reporter=dot,list
+          NEXTAUTH_SECRET=SECRET npm start & 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)